atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/transaction/knowledge/ConcludedRangeStateTest.java

```diff
@@ -130,4 +134,61 @@ public void copyAndAddDoesNotModifyLowerBoundOnPreExistingRanges() {
         Range<Long> range = Range.atLeast(upperbound + 1);
         assertThat(state.copyAndAdd(range).timestampRanges().asRanges()).containsExactly(initialRange, range);
     }
+
+    @Test
+    public void copyAndAddDoesNotAddRangesWhichAreBelowMinimumTimestamp() {
+        long minimumTimestamp = 50l;
```

Comment:
is this test not same as `copyAndAddDoesNotAddRangeIfIntersectionIsEmpty`

Reply:
Slight difference! `copyAndAddDoesNotAddRangesWhichAreBelowMinimumTimestamp` doesn't do a boundary test, while `copyAndAddDoesNotAddRangeIfIntersectionIsEmpty` does. I thought it was different enough to warrant different tests (i.e. an empty range due to intersection is different than no intersection).